debian/changelog

```diff
@@ -1,3 +1,13 @@
+regolith-wm-config (4.11.9) jammy; urgency=medium
```

Comment:
nit: I prefer we don't update `d/changelog` file manually on PRs and use [Update Package Changelog](https://github.com/regolith-linux/voulage/actions/workflows/update-changelog.yml) when the time is right to update it. Although it's really just a preference.

Reply:
Sure, so do I just remove this file, `debian/changelog`, from the PR?  Anything else I need to do to cause the additional entry to be added?